### DISCUSSION TITLE
Unable to Override Command + H on macOS

### DISCUSSION THREAD

**Discussion by LandonSchropp** [ghostty-org/ghostty#4213] [Feature Requests, Ideas] [answered]
I'm attempting to port a couple of keybindings for command + h and command + l from my iTerm configuration over to Ghostty. The command + l keybinding works just fine, but command + h hides the window instead of triggering my keybinding. 

Here's what my configuration looks like:

```
keybind = super+h=text:\x13p
keybind = super+l=text:\x13n
```

It seems that even though I've set the `super+h` binding, the system hide shortcut is overriding it. This can be seen in Ghostty's menu:

![Screenshot 2024-12-31 at 11 08 23 AM](https://github.com/user-attachments/assets/0c1b15b5-9298-48c5-912a-9ec6757ee70e)

This is likely related to #4134, but I'm differentiating it since my version of this issue involves new keybindings being overridden rather than clearing existing bindings.

Thank you!

**Comment by rhodes-b** [comment, marked as answer]
I actually think this is a duplicate of https://github.com/ghostty-org/ghostty/discussions/3187

**Reply by LandonSchropp** [reply]
@rhodes-b Yep, I think you're right. Thank you!
